packages/wallet/backend/src/cardUsers.ts

```diff
@@ -0,0 +1,110 @@
+import { createContainer } from '@/createContainer'
+import { env } from '@/config/env'
+import { ICreateCustomerRequest } from '@/card/types'
+
+interface UserData {
+  email: string
+  firstName: string
+  lastName: string
+  ppNumber: string
+}
+
+const entries = `John;Doe;john@doe.com:8888
```

Comment:
is this for example porposes? can the list be read from an external file?

Reply:
This is how the entries should be added. It can be updated to read from an external file as well, but for simplicity reason we decided to just put the CSV contents directly in the variable.